addon/styles/ui-tomato-theme.scss

```diff
@@ -1,2 +1,3 @@
-// TODO import ui-base-theme once the preprocessors will allow it
+@import 'ui-base-theme/ui-base-theme';
```

Comment:
I'm not sure why this path needs to be used rather than just `@import "ui-base-theme";`